phone_book.py

```diff
@@ -0,0 +1,35 @@
+book = dict()
+a = True
+while a:
+    command_user = input("Введіть команду (stats, add, delete, list, show, exit): ")
+    if command_user.lower() == "stats":
```

Comment:
nit: too many duplicates of the `lower()`.
```suggestion
    command_user_lower = command_user.lower() 
    if command_user_lower == "stats":
```